apps/vr-tests/src/stories/Button.stories.tsx

```diff
@@ -9,7 +9,7 @@ const baseProps: IButtonProps = {
   iconProps: {
     iconName: 'AddFriend'
   },
-  children: 'Butjon',
+  children: 'Button',
```

Comment:
The `j` is on purpose to test descender characters don't exceed the boundary.

Perhaps we should use something more obviously a test string? Like 'Button ABC123jxyz'